content/docs/rfcs/13/README.md

```diff
@@ -51,14 +62,18 @@ message PagingInfo {
   Direction direction = 3;
 }
 
+message HistoryContentFilter {
+  string contentTopic = 1;
+}
+
 message HistoryQuery {
-  repeated string topics = 2;
-  optional PagingInfo pagingInfo = 3; // used for pagination
+  repeated HistoryContentFilter contentFilters = 2;
```

Comment:
Some of the types are prefixed with `History` and some are not (`PagingInfo`), what is the reasoning here?

Reply:
I am not sure about the exact reason but to my understanding, since the `Request`s and `RPC`s message names are shared with the filter protocol, the `History` prefix (and similarly the `Filter` prefix) would disambiguate the usage.  The same applies to the `ContentFilter` which is used with a similar name in the filter protocol.
However, `PagingInfo` is only used in the store protocol so is not prefixed.

Do you have any suggestion re this? let me know!

Reply:
Sounds reasonable 👍